docker/nginx/conf.d/client.conf

```
@@ -25,6 +25,10 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
+	# parse subdomain into custom variable
+	server_name ~^([a-z0-9]{55}).*?\.siasky\.(dev|net)$;
```

Would this work ? A little bit more generic.

```suggestion
	server_name ~^([a-z0-9]{55}).*?;
```